src/main/kotlin/com/launchdarkly/intellij/toolwindow/LinkNodeRoot.kt

```diff
@@ -0,0 +1,83 @@
+package com.launchdarkly.intellij.toolwindow
+
+import com.intellij.ide.projectView.PresentationData
+import com.intellij.ui.treeStructure.SimpleNode
+import com.launchdarkly.intellij.settings.LDSettings
+
+class LinkNodeRoot(private val settings: LDSettings) :
+    SimpleNode() {
+    private var myChildren: MutableList<SimpleNode> = ArrayList()
+
+    override fun getChildren(): Array<SimpleNode> {
+        if (settings.project != "" && settings.environment != "") {
+            myChildren.add(
```

Comment:
Curious why Account Settings is not included here? It can be useful to check permissions/api keys.

Reply:
I felt like that was a bit too much for the links as that's really the only time it was used but I'm open to adding it if we want it there.